SoSe16/Bericht_SoSe16_Konstanz.tex

```diff
@@ -20,13 +20,16 @@
 \definecolor{urlred}{HTML}{660000}
 \usepackage{hyperref}
 \hypersetup{
-      colorlinks=true,
-      linkcolor=black,                              % Farbe der internen Links (u.a. Table of Contents)
-      urlcolor=urlred}                              % Farbe der url-links
+	colorlinks=true,
+	linkcolor=black,                              % Farbe der internen Links 
+	%(u.a. Table of Contents)
+	urlcolor=urlred}                              % Farbe der url-links
 \parindent 0pt                                      % Absatzeinrücken verhindern
-\parskip 12pt                                       % Absätze durch Lücke trennen
+\parskip 12pt                                       % Absätze durch Lücke 
+%trennen
```

Comment:
Hmm, am einfachsten wäre es wahrscheinlich, wenn Du Deine jetzigen Änderungen erst in einem anderen Branch speicherst:
```
git branch <irgendein toller name>
```
Danach setzt Du deinen Git-Master auf die Ursprüngliche Version zurück:
```
git reset --hard 237c60b
```
(die Version siehst du entweder im `git log`, oder du nutzt die elegantere Version, indem Du Dir ein upstream Remote einrichtest, und dann auf upstream/master resettest).
Danach bearbeitest Du die Datei, und speicherst sie mit den 80 Zeichen Spaltenbreite ab, lässt sie allerdings inhaltlich ungeändert, und commitest das.
Danach kannst Du mit `git cherry -v master <dein toller name für den anderen branch>` Dir die Commit ID's für deine anderen Commits ansehen.
Dann kannst du die Änderungen der Dateien in dem anderen Branch auf deinen neuen Commit anwenden:
```
git checkout 7ea70393e7dbb65d1c80fd79140f976303c470e3 SoSe16/Bericht_SoSe16_Konstanz.tex SoSe16/bericht_data.tex
```
Danach neu kompilieren, adden, neu committen.
Das gleiche nochmal für den 2. Commit:
```
git checkout 9cba85cfcbebfb234d0e9fc4a6ae667752357b1a SoSe16/Bericht_SoSe16_Konstanz.tex SoSe16/bericht_data.tex
```
Das ganze kannst du dann force-pushen:
```
git push -f
```